README.md

```diff
@@ -265,10 +265,14 @@ Structure similarity queries also allow users to upload a file from their local
 ```python
 from rcsbsearchapi.search import StructSimilarityQuery
 
-q3 = StructSimilarityQuery("file_url", "https://files.rcsb.org/view/4HHB.cif", input_option="cif")
+q3 = StructSimilarityQuery(structure_search_type="file_url",
+                           file_url="https://files.rcsb.org/view/4HHB.cif",
+                           file_format="cif")
 
 # If using file upload, an example query would be like below:
-# q3 = StructSimilarityQuery("file_upload", "/absolute/path/to/file.cif", input_option="file format")
+# q3 = StructSimilarityQuery(structure_search_type="file_upload",
+#                            file_upload="/absolute/path/to/file.cif",
+#                            file_format="file format")
 
 list(q3())
```

Comment:
I think this will need to change to `file_path` after the change I suggested earlier. Also, `file_format="file_format"` doesn't look quite right...

That said, since you'll need to change this anyway, can you replace these lines with the following:
```python
q3 = StructSimilarityQuery(structure_search_type="file_url",
                           file_url="https://files.rcsb.org/view/4HHB.cif",
                           file_format="cif")
list(q3())

# If you want to upload your own structure file for similarity search, you can do so by using the `file_path` parameter:
q4 = StructSimilarityQuery(structure_search_type="file_upload",
                           file_path="/path/to/file.cif",  # specify local model file path
                           file_format="cif")
list(q4())
```

I think this helps emphasize the file-upload capability a little bit more. 

Apologies for the redundant changes that will need to be made in the other `.md` files though...

Reply:
no worries!